File: week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // prompt user for height of pyramid
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0);

    // loop to print pyramid
    for (int i = 0; i < height; i++)
    {
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    # checks if user input is greater than 0 and less sthan 8
    if height > 0 and height < 9:
        break

for i in range(height):
    # prints spaces
    for j in range(height - i - 1):
        print(" ", end="")
    # prints '#' after spaces
    for k in range(i + 1):
        print("#", end="")
    print()
